src/mono/mono/mini/debugger-engine.c

```diff
@@ -1754,5 +1754,22 @@ get_notify_debugger_of_wait_completion_method (void)
 	return notify_debugger_of_wait_completion_method_cache;
 }
 
+void
+mono_de_set_interp_var (MonoType *t, gpointer addr, guint8 *val_buf)
+{
+	int size;
+
+	if (t->byref) {
+		addr = *(gpointer*)addr;
+		g_assert (addr);
```

Comment:
should we return `FALSE` here, instead, and fail gracefully?

Reply:
or if we want to assert for the non-wasm cases, then add a `mono_de_try_set_interp_var` for use with wasm?

Reply:
Done.